AUTHORS

```diff
@@ -898,10 +900,11 @@ zoubin <zoubin04@gmail.com>
 Robert Jefe Lindstaedt <robert.lindstaedt@gmail.com>
 Saquib <saquibofficial@gmail.com>
 Daniel Sellers <daniel@designfrontier.net>
-Julien Waechter <julien.waechter@gmail.com>
+julien.waechter <julien.waechter@gmail.com>
```

Comment:
nope... the update-authors tool changed a bunch of these. I'm changing them back